src/kOS/Screen/TermWindow.cs

```diff
@@ -612,18 +724,18 @@ protected void CheckResizeDrag()
             }
         }
         
-        void ShowCharacterByAscii(char ch, int x, int y, Color charTextColor)
+        void ShowCharacterByAscii(char ch, int x, int y, Color charTextColor, bool reversingScreen)
         {
             int tx = ch % FONTIMAGE_CHARS_PER_ROW;
             int ty = ch / FONTIMAGE_CHARS_PER_ROW;
 
-            ShowCharacterByXY(x, y, tx, ty, charTextColor);
+            ShowCharacterByXY(x, y, tx, ty, charTextColor, reversingScreen);
         }
 
-        void ShowCharacterByXY(int x, int y, int tx, int ty, Color charTextColor)
+        void ShowCharacterByXY(int x, int y, int tx, int ty, Color charTextColor, bool reversingScreen)
         {
             GUI.BeginGroup(new Rect((x * CHARSIZE), (y * CHARSIZE), CHARSIZE, CHARSIZE));
-            GUI.color = charTextColor;
+            GUI.color = (reversingScreen ? bgColor : charTextColor);
             GUI.DrawTexture(new Rect(tx * -CHARSIZE, ty * -CHARSIZE, fontImage.width, fontImage.height), fontImage);
```

Comment:
This line is very inefficient looking and I'd like to address fixing it - but given then KSP is moving to Unity 5 next, I'm reluctant to mess with it now when everything about the GUI system is getting an overhaul anyway.

What this line does is paint one character out of the fontImage by positioning the font image's upper corner at a negative coordinate relative to the origin of the tiny 8x8 rectangle this BeginGroup is cropping to, such that the little 8x8 rectangle of the BeginGroup will only allow one of the characters of the fontImage to "bleed through" to be seen, cropping off the rest of it.  It still draws the whole fontImage, and relys on the BeginGroup to crop off the unseen parts of it.  It then repeats this over and over for each letter it prints.  It seems to me that it would be more efficient to pull a subrectangle for one of the characters out of the fontImage and paint just that, rather than painting the whole fontImage and relying on the cropping of the output rectangle to hide the parts of the fontimage outside of the desired letter.